src/api/server/common.rst

```diff
@@ -728,6 +734,10 @@
     :json number doc_write_failures: Number of document write failures
     :json number docs_read:  Number of documents read
     :json number docs_written:  Number of documents written to target
+    :json number bulk_get_attempts: The count of attempted doc fetches
+        with ``_bulk_get``.
```

Comment:
Is this the number of times `_bulk_get` is called, or the number of docs fetched with `_bulk_get`?  I suspect the former, but it might be nice to make that a bit more explicit.

Reply:
It's the later to match the docs_read/docs_written and counts the number of docs revisions fetched with _bulk_get.

Reply:
Updated with the `total` wording.